### PR TITLE
[REVIEW] Fix flake8 issues

### PR DESCRIPTION
**Pull request by kkraus14**

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,15 +4,19 @@ repos:
         hooks:
               - id: isort
       - repo: https://github.com/ambv/black
-        rev: stable
+        rev: 19.10b0
```
Comment (kkraus14) [inline .pre-commit-config.yaml:7]: Revert back to stable

File: python/.flake8 [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
```
Comment (kkraus14) [inline python/.flake8:1]: Fix copyright year

File: python/setup.py [hunk 3]
```diff
@@ -27,7 +27,7 @@
 
 if not os.path.isdir(CUDA_HOME):
     raise OSError(
-        f"Invalid CUDA_HOME: " "directory does not exist: {CUDA_HOME}"
+        "Invalid CUDA_HOME: " f"directory does not exist: {CUDA_HOME}"
```
Comment (kkraus14) [inline python/setup.py:30]: Merge this into a single string

**Pull request merged**